packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterInputMultipleValue.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Autocomplete, { AutocompleteProps } from '@mui/material/Autocomplete';
+import Chip from '@mui/material/Chip';
+
+import TextField from '@mui/material/TextField';
+import { unstable_useId as useId } from '@mui/material/utils';
+import { GridFilterItem } from '../../../models/gridFilterItem';
+
+export type GridFilterInputMultipleValueProps = {
+  item: GridFilterItem;
+  applyValue: (value: GridFilterItem) => void;
+  // Is any because if typed as GridApiRef a dep cycle occurs. Same happens if ApiContext is used.
+  apiRef: any;
+  focusElementRef?: React.Ref<any>;
+  type?: 'text' | 'number';
+} & Omit<AutocompleteProps<any[], true, false, true>, 'options' | 'renderInput'>;
+
+function GridFilterInputMultipleValue(props: GridFilterInputMultipleValueProps) {
+  const { item, applyValue, type, apiRef, focusElementRef, ...other } = props;
+  const [filterValueState, setFilterValueState] = React.useState(item.value || []);
+  const id = useId();
+
+  React.useEffect(() => {
+    const itemValue = item.value ?? [];
+    setFilterValueState(itemValue.map(String));
+  }, [item.value]);
+
+  const onFilterChange = React.useCallback(
+    (event, value) => {
+      const parsedValue = value.map(String);
```

Comment:
Why to convert to string?

Reply:
Because the original component was converting value to string before doing the update, so I applied the same logic on the multipleValue component.

https://github.com/mui-org/material-ui-x/blob/61e27becdc716a27205d5afda19d0e0c3dbd94de/packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterInputSingleSelect.tsx#L56-L67

Reply:
I did not found any interest in doing it, so I removed all the String() from my code